### PR TITLE
GEOPY-1099

### PR DESCRIPTION
**Pull request by benk-mira**

### CODE REVIEW

File: las_geoh5/import_files/driver.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+from las_geoh5.import_las import LASTranslator, las_to_drillhole
+
+
+def run(file: str):
+    ifile = InputFile.read_ui_json(file)
+
+    # TODO: Once fix implemented in geohh5py can revert back to simply pulling
+    # drillhole group from input file rather that using get_entity.
+    # dh_group = ifile.data["drillhole_group"]
```
Comment (domfournier) [inline las_geoh5/import_files/driver.py:24]: Do we have an issue for this?

File: las_geoh5/import_las.py [hunk 2]
```diff
@@ -46,23 +105,34 @@ def get_depths(lasfile: lasio.LASFile) -> dict[str, np.ndarray]:
...
-    for attr in ["X", "Y", "ELEV"]:
-        if hasattr(lasfile.well, attr):
-            collar.append(lasfile.well[attr].value)
+    for field in ["collar_x", "collar_y", "collar_z"]:
+        try:
+            collar.append(translator.retrieve(field, lasfile))
+        except KeyError:
+            warnings.warn(
+                f"{field.replace('_', ' ').capitalize()} field "
+                f"'{getattr(translator, field)}' not found in las file."
```
Comment (domfournier) [inline las_geoh5/import_las.py:129]: We were thinking providing the list of fields available in file to help the user correct the inputs.

**Review by domfournier — COMMENTED** [review]
Looks good. Small comment about the print message.

File: las_geoh5/import_files/driver.py (hunk 1 above)
Comment (benk-mira) [reply las_geoh5/import_files/driver.py:24]: GEOPY-1111

File: las_geoh5/import_las.py (hunk 2 above)
Comment (benk-mira) [reply las_geoh5/import_las.py:135]: done

File: las_geoh5/import_files/driver.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    files = ifile.data["files"].split(";")
+    files = [lasio.read(file, mnemonic_case="preserve") for file in files]
+    translator = LASTranslator(
+        depth=ifile.data["depths_name"],
+        collar_x=ifile.data["collar_x_name"],
+        collar_y=ifile.data["collar_y_name"],
+        collar_z=ifile.data["collar_z_name"],
+    )
+    with fetch_active_workspace(ifile.data["geoh5"], mode="a") as workspace:
+        dh_group = ifile.workspace.get_entity(ifile.data["drillhole_group"].name)[0]
```
Comment (domfournier) [inline las_geoh5/import_files/driver.py:36]:
```suggestion
        dh_group = ifile.workspace.get_entity(ifile.data["drillhole_group"].uid)[0]

```

**Review by domfournier — APPROVED** [review]

**Pull request merged**